pkgs/apps/openmolcas/v21.02.nix

```diff
@@ -1,11 +1,20 @@
 { lib, stdenv, pkgs, fetchFromGitLab, fetchpatch, cmake, gfortran, perl
-, openblas, hdf5-full, python3, texlive
+, blas-i8, hdf5-full, python3, texlive
 , armadillo, makeWrapper, fetchFromGitHub, chemps2
 } :
 
+assert
+  lib.asserts.assertMsg
+  (blas-i8.isILP64 || blas-i8.passthru.implementation == "mkl")
+  "A 64 bint integer BLAS implementation is required.";
+
+assert
+  lib.asserts.assertMsg
+  (builtins.elem blas-i8.passthru.implementation [ "openblas" "mkl" ])
+  "OpenMolcas requires OpenBLAS or MKL.";
+
 let
   version = "21.02";
```

Comment:
I wonder if we should also append a date string to the version (`.YYYY-MM-DD`) to bring more clarity into the inconsistent versioning?

Reply:
Might be a good idea. I guess they are using their moving tags just for minor patch updates (at least from what I've observed so far). I guess nothing should change results there, but at least it changed it from "does not build with this feature"  to "builds with this feature" a few times ...

Reply:
Then let's add the date strings to the version. This brings in more clarity. Ideally the molcas developer would create a proper release with patchlevels (i.e. 21.02.X) but since this is not the case we need to work around it.